Bots/DotNet/Skills/CodeFirst/TeamsWaterfallSkillBot/Properties/launchSettings.json

```diff
@@ -1,9 +1,10 @@
 {
+  "$schema": "http://json.schemastore.org/launchsettings.json",
   "iisSettings": {
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:39820/",
+      "applicationUrl": "http://localhost:35420/",
```

Comment:
This port is used by the WaterfallSkillBot, let's use a different one for this one (not sure if we will still need it)

Reply:
updated.